test/integration/dra/dra_test.go

```diff
@@ -573,12 +574,14 @@ func testPrioritizedList(tCtx ktesting.TContext, enabled bool) {
 	}).WithTimeout(10 * time.Second).WithPolling(time.Second).Should(schedulingAttempted)
 }
 
-func testPublishResourceSlices(tCtx ktesting.TContext, disabledFeatures ...featuregate.Feature) {
+func testPublishResourceSlices(tCtx ktesting.TContext, haveLatestAPI bool, disabledFeatures ...featuregate.Feature) {
```

Comment:
We don't have event handler tests in the unit tests of the package. Testing work queue behavior is tricky there and there's always the question whether the implemented behavior works "in the real world".

Therefore I added an integration test instead. I also used the opportunity to tests the "update slice after unwanted update" scenario.

Both tests need the v1beta2 API, whereas the original "create slices" doesn't.